config/secrets.yml

```diff
@@ -43,8 +43,7 @@ default: &default
       client_id: <%= ENV["OMNIAUTH_GOOGLE_CLIENT_ID"] %>
       client_secret: <%= ENV["OMNIAUTH_GOOGLE_CLIENT_SECRET"] %>
   geocoder:
-    here_app_id: <%= ENV["GEOCODER_LOOKUP_APP_ID"] %>
-    here_app_code: <%= ENV["GEOCODER_LOOKUP_APP_CODE"] %>
+    google_geocoder_api_key: <%= ENV["GOOGLE_GEOCODER_API_KEY"] %>
```

Comment:
Will we need this key locally for testing purposes?

Reply:
Yeah. You can use the key in the heroku config settings: https://dashboard.heroku.com/apps/decidim-seattle-staging/settings